jsapp/js/formbuild/containers/KoboMatrix.es6

```diff
@@ -263,12 +348,12 @@ class KoboMatrix extends React.Component {
 
   getRequiredStatus(colKuid) {
     const val = this.state.data.getIn([colKuid, 'required']);
-    return val == true || val == 'true' ? true : false;
+    return val === true || val === 'true' ? true : false;
   }
 
   newChoiceOption(e) {
-    var data = this.state.data,
-        listName = null;
+    var data = this.state.data;
+    let listName = null;
```

Comment:
Is there a reason to have both `let` and `var` here (considering you've now changed that line anyway 💁‍♂️

Reply:
Right, fixed :)